.travis.yml

```diff
@@ -25,11 +14,13 @@ env:
     - DB_NAME=${TRAVIS_JOB_ID}
 
 before_install:
-  - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
   - pip install acsoo
   - pip install codecov
-  - for addon in $(ls setup/ -I README) ; do echo "-e ./setup/$addon" >> requirements.txt; done
+  - ADDONS_INST=$(acsoo addons --addons-dir ${TRAVIS_BUILD_DIR} list-depends)
+  - ADDONS_TEST=$(acsoo addons --addons-dir ${TRAVIS_BUILD_DIR} list)
+  - ADDONS_TEST_LIST=$(echo $ADDONS_TEST | tr "," "\n")
```

Comment:
Ah, indeed. Parhaps `acsoo addons` should grow a `--separator` option.

Reply:
yes. It would be nice